src/Command/DbDump.php

```diff
@@ -135,8 +147,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->maintenanceModeSwitcher->enable();
             $this->cronSwitcher->disable();
             $this->backgroundProcess->kill();
-            $this->dumpGenerator->create((bool)$input->getOption(self::OPTION_REMOVE_DEFINERS));
-            $this->logger->info('Backup completed.');
```

Comment:
Why did you remove info log about backup completion?

Reply:
It seems like an error occurred while resolving conflicts. I returned the string back